podcasts/Whats New/Announcements.swift

```diff
@@ -19,6 +22,45 @@ struct Announcements {
 
                 NavigationManager.sharedManager.navigateTo(NavigationManager.settingsProfileKey, data: nil)
             }
+        ),
+
+        // Bookmarks Early Access: Beta
+        // Show only in TestFlight, for Plus and Patron
+        .init(
+            version: "99.99",
+            header: { AnyView(EmptyView()) },
+            title: "Early Access Beta Title",
+            message: "Message",
+            buttonTitle: "Button",
+            action: {},
+            isEnabled: PaidFeature.bookmarks.inEarlyAccess && BuildEnvironment.current == .testFlight && SubscriptionHelper.activeTier > .none
```

Comment:
The `isEnabled` is a `Bool` that is calculated when the `Announcement` is created. As it is right now, I think this is pretty quick and straightforward. However, I'm thinking about the future: what if we stack many announcements with special conditions? That would mean that this logic will always run when the app starts and I wonder if we want that?

If we make that an `@autoclosure` we can eliminate this problem.

No need to do in this PR right now but something to keep in mind.

Reply:
I can see that, I updated to use an autoclosure. Also since I was defining it, I also updated the header view to use it as well. 

6d017035c13f5c3214c2a80f7f59608f51e97eef

Reply:
> I also updated the header view to use it as well.

Nice!